assets/app/view/game_page.rb

```diff
@@ -81,6 +81,8 @@ def render
           h(Game::Companies, game: @game, user: @user)
         when 'corporations'
```

Comment:
get rid of the other tabs

Reply:
Tabs are long gone, these are just the route anchors. I’d leave them in place as I expect the need for them later when people clamour for sorting options … but we could of course revert then.
If you want the code to be pruned, just say so. There are more unnecessary hunks due to entities. (Maybe transitional, maybe for good – depends on sorting yes/no.)